app/api/events.py

```diff
@@ -63,6 +63,15 @@ class Meta:
                            schema='SponsorSchema',
                            many=True,
                            type_='sponsor')
+    notification = Relationship(
+        attribute='notification',
+        self_view='v1.event_notification',
+        self_view_kwargs={'id': '<id>'},
+        related_view='v1.notification_detail',
```

Comment:
I am not sure if notification_list should be there or notification_detail. In the model event has relation only to get notification email. What do you think how this relationship should be defined?